pkg/apis/kubedirector/v1beta1/kubedirectorcluster_types.go

```diff
@@ -19,6 +19,16 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	// UID represents the old naming scheme where object names were generated
+	// with unique UID extensions.
+	UID string = "UID"
+
+	// CrNameRole represents the new naming scheme based on cluster name and
+	// respective role name.
+	CrNameRole string = "CrNameRole"
```

Comment:
To utilize the new naming, will they have to put this string "CrNameRole" in their spec ?

Reply:
Yes!!